improver/spotdata/build_spotdata_cube.py

```diff
@@ -68,7 +68,7 @@ def build_spotdata_cube(
        build_spotdata_cube_examples.rst
 
     Args:
-        data (float or numpy.ndarray):
+        data (numpy.ndarray):
```

Comment:
If we require this to be a numpy array, the doc-string needs to be updated as well.